### PR TITLE
Output magento2 reports into the container logs

### PR DESCRIPTION
**Pull request by kierenevans**
Every minute, find the report files written out to /app/var/report/ and print them out to the container logs so they are discoverable.
Depends on #297 for the do_magento_echo_last_minute_reports function that is being called.

### CODE REVIEW

**Comment by kierenevans** [comment]
The do_magento_echo_last_minute_reports function will be present in the magento_functions.sh location with if https://github.com/continuouspipe/dockerfiles/pull/297/files#diff-4a68f150fec1a766024815e12719d4f0R373 is merged in.

File: magento2-nginx/fpm-7.0/etc/confd/templates/magento/echo_reports_cron.tmpl [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# don't send any mail
+MAILTO=""
+SUPERVISORCTL="supervisorctl -c /etc/supervisor/supervisord.conf -u supervisor -p supervisor"
+
+* * * * * root $SUPERVISORCTL start echo_magento_reports
```
Comment (sroze) [inline magento2-nginx/fpm-7.0/etc/confd/templates/magento/echo_reports_cron.tmpl]: Shouldn't we be able to disable this?

File: magento2-nginx/fpm-7.0/etc/confd/templates/magento/cron.tmpl [hunk 2]
```diff
@@ -2,4 +2,6 @@
 MAILTO=""
 SUPERVISORCTL="supervisorctl -c /etc/supervisor/supervisord.conf -u supervisor -p supervisor"
 
+{{ if eq "true" (getenv "RUN_MAGENTO_CRON") }}
```
Comment (sroze) [inline magento2-nginx/fpm-7.0/etc/confd/templates/magento/cron.tmpl]: Not sure but is this environment variable documented?

File: magento2-nginx/fpm-7.0/etc/confd/templates/magento/echo_reports_cron.tmpl (hunk 1 above)
Comment (kierenevans) [inline magento2-nginx/fpm-7.0/etc/confd/templates/magento/echo_reports_cron.tmpl]: Added a variable for it 👍

File: magento2-nginx/fpm-7.0/etc/confd/templates/magento/cron.tmpl (hunk 2 above)
Comment (kierenevans) [inline magento2-nginx/fpm-7.0/etc/confd/templates/magento/cron.tmpl]: It is now! Good spot

**Pull request merged**